src/components/dump.py

```diff
@@ -615,7 +615,24 @@ def find(self, packet_type=None):
             return next(self.fetch(packet_type))
         except StopIteration:
             return None
-    
+
+    def findLast(self, packet_filter=()):
+        try:
+            iter(packet_filter)
+        except TypeError:
+            packet_filter = (packet_filter,)
+        else:
+            packet_filter = packet_filter
+
+        # Append new packets
+        for dump in self.__dumps.fetch():
```

Comment:
Why is this function not fetching in a loop like the `fetch` function?

Reply:
Performance mostly. Looping through all the packages several hundred times just to get the last packet had a noticeable impact on run-time.
In addition, it is also slightly easier to use than having a loop.